docs/cumulusci_flow.rst

```diff
@@ -0,0 +1,273 @@
+CumulusCI Flow
```

Comment:
Overarching comment: the narrative voice is not fully consistent here. In part, we speak to "you" about your capabilities and resources - "you can" or "you should". But in part, we speak in "we do", as a description of our operations. 

I would suggest that this section may be most effective if the framing is "This is what CumulusCI Flow is.  We find this value in it. Here's how it works." That is, we front-load the impetus and benefits of the process, and then speak in descriptive terms about the process, rather than about what we actually do.